src/pages/AddPersonalBankAccountPage.js

```diff
@@ -163,7 +122,14 @@ class AddPersonalBankAccountPage extends React.Component {
                         </FixedFooter>
                     </>
                 ) : (
-                    <FormScrollView>
+                    <Form
+                        formID={ONYXKEYS.PERSONAL_BANK_ACCOUNT}
```

Comment:
I know this already exists, but I'm hazy on how the form IDs work - would it make more sense to add a new `ONYXKEYS.FORMS.PERSONAL_BANK_ACCOUNT` to separate it out from the key for the value itself?

Otherwise this is looking good when I skimmed; I will look closer and also test tomorrow!

Reply:
Yes one of the reasons I didn't wanna move the key under Forms is that there are other external PRs ongoing that touch the same keys so we figured out that we'll keep keys as they are now and then once everything is merge we'll move the keys under the `FORMS` key. But I'm gonna double check in case this specific key is only used here then I can move it under `FORMS`. Good eye!